htdocs/class/xoopsform/formselect.php

```diff
@@ -216,7 +220,12 @@ public function render()
             }
             $ret .= '>' . $name . '</option>';
         }
-        $ret .= '</select>';
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $ret .= '</select></div>';
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->